### PR TITLE
Update outfile(en) to support cos.md

### PR DESCRIPTION
**Pull request by BabySid**
update doc to add how to export query result on cos

## Proposed changes

Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.

## Types of changes

What types of changes does your code introduce to Doris?
_Put an `x` in the boxes that apply_

- [] Bugfix (non-breaking change which fixes an issue)
- [] New feature (non-breaking change which adds functionality)
- [] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [] Documentation Update (if none of the other choices apply)
- [] Code refactor (Modify the code structure, format the code, etc...)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [] I have create an issue on (Fix #ISSUE), and have described the bug/feature there in detail
- [] Compiling and unit tests pass locally with my changes
- [] I have added tests that prove my fix is effective or that my feature works
- [] If this change need a document change, I have updated the document
- [] Any dependent changes have been merged

## Further comments

If this is a relatively large or complex change, kick off the discussion at dev@doris.apache.org by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Review by EmmyMiao87 — DISMISSED** [review]
LGTM

File: docs/en/administrator-guide/outfile.md [hunk 1]
```diff
@@ -156,7 +156,35 @@ WITH BROKER `broker_name`
...
+       "line_delimiter" = "\n",
+       "max_file_size" = "1024MB"
+    )
+    ```
+    
+    If the result is less than 1GB, file will be: `my_file_0.csv`.
     
+    If larger than 1GB, may be: `my_file_0.csv, result_1.csv, ...`.
+    
+    Some notes:
```
Comment (wuyunfeng) [inline docs/en/administrator-guide/outfile.md:184]: Notes就行了 别这么硬

File: docs/en/administrator-guide/outfile.md [hunk 2]
```diff
@@ -156,7 +156,35 @@ WITH BROKER `broker_name`
...
+    )
+    ```
+    
+    If the result is less than 1GB, file will be: `my_file_0.csv`.
     
+    If larger than 1GB, may be: `my_file_0.csv, result_1.csv, ...`.
+    
+    Some notes: 
+    1. Paths that do not exist are automatically created.
+    2. These parameters(access.key/secret.key/endpointneed) need to be confirmed with cos. In particular, the value of endpoint does not need to be filled in bucket_name。
```
Comment (wuyunfeng) [inline docs/en/administrator-guide/outfile.md:186]:
```suggestion
    2. These parameters(access.key/secret.key/endpointneed) need to be confirmed with `Tecent Cloud COS`. In particular, the value of endpoint does not need to be filled in bucket_name。
```

**Review by wuyunfeng — CHANGES_REQUESTED** [review]
Great. And I left some small comment

File: docs/en/administrator-guide/outfile.md (hunk 1 above)
Comment (wuyunfeng) [inline docs/en/administrator-guide/outfile.md:184]:
```suggestion
    Please Note: 
```

Comment (blackfox1983) [reply docs/en/administrator-guide/outfile.md]: 帮我直接comments即可。我这边可以直接commit

**Review by wuyunfeng — APPROVED** [review]

**Pull request merged**